packages/synapse-client/tsconfig.build.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "noEmit": false,
+    "types": ["vitest/globals"]
+  },
+  "include": ["src/*.ts", "src/generated/**/*.ts", "src/util/**/*.ts"],
```

Comment:
This will be built with ts, but don't emit the `src/codegen` folder, which are internal utilities that other packages should not use